src/ImageSharp/Formats/Webp/Lossy/YuvConversion.cs

```diff
@@ -312,6 +586,178 @@ public static void YuvToBgr(int y, int u, int v, Span<byte> bgr)
             bgr[0] = (byte)YuvToB(y, u);
         }
 
+#if SUPPORTS_RUNTIME_INTRINSICS
+
+        private static void ConvertYuvToBgrSse41(Span<byte> topY, Span<byte> bottomY, Span<byte> topDst, Span<byte> bottomDst, Span<byte> ru, Span<byte> rv, int curX, int step)
+        {
+            YuvToBgrSse41(topY.Slice(curX), ru, rv, topDst.Slice(curX * step));
+
+            if (bottomY != null)
```

Comment:
This if can be moved outside of the loop for happier pipelines / branch predictor.

Reply:
Moved the if outside of the loop with 65870b96